client/src/client-only-routes/show-certification.tsx

```diff
@@ -267,6 +267,7 @@ const ShowCertification = (props: ShowCertificationProps): JSX.Element => {
           />
         </Col>
       </Row>
+      <Spacer size='medium' />
```

Comment:
Is this intentional? It seemed a little out of scope.

Reply:
Yes, I moved the space that was under the form to the close button.